.github/workflows/test_deploy.yml

```diff
@@ -19,6 +20,11 @@ on:
         required: true
         default: 'Bluedot-React-Native-Plugin release'
 
+# Some global environments to avoid duplicates
+env:
+  ANDROID_SIGNED_NAME: "app-release-signed.apk"
+  ANDROID_RELEASE_DIR: "android/app/build/outputs/apk/release"
```

Comment:
centralised some env variables to avoid duplicating + standardising (signed) output name